.github/workflows/tests.yml

```diff
@@ -31,6 +31,18 @@ jobs:
       image: ghcr.io/homebrew/ubuntu22.04:master
     env:
       HOMEBREW_SIMULATE_MACOS_ON_LINUX: 1
+    steps:
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - run: brew test-bot --only-tap-syntax
+
+  formulae_detect:
+    if: github.repository == 'Homebrew/homebrew-core' && github.event_name == 'pull_request'
```

Comment:
Note that I think we should think sooner rather than later about having the `merge_group` job run `brew fetch` for these formulae that are changed so I think it's worth continuing to run this.

Reply:
I've added a `brew test-bot --only-formulae-detect` step back to `tap_syntax`.

Reply:
@carlocab Thanks 🙇🏻